apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/EstimatedExecutionPrice.tsx

```diff
@@ -139,19 +186,34 @@ export function EstimatedExecutionPrice(props: EstimatedExecutionPriceProps) {
           wrapInContainer={true}
           content={
             <styledEl.ExecuteInformationTooltip>
-              {isNegativeDifference && Math.abs(Number(percentageDifferenceInverted?.toFixed(4) ?? 0)) <= 0.01 ? (
-                <>Will execute soon!</>
+              {isNegativeDifference &&
+              Math.abs(Number(percentageDifferenceInverted?.toFixed(4) ?? 0)) <=
```

Comment:
`Math.abs(Number(percentageDifferenceInverted?.toFixed(4) ?? 0)) <=
                PENDING_EXECUTION_THRESHOLD_PERCENTAGE` should be defined to a separate const with description